cordova-lib/cordova-lib.js

```diff
@@ -18,19 +18,35 @@
 */
 
 // For now expose plugman and cordova just as they were in the old repos
+
+
+function addProperty(obj, property, modulePath) {
```

Comment:
Similar functionality already exists at [`src/cordova/util.js:339`](https://github.com/apache/cordova-lib/blob/master/cordova-lib/src/cordova/util.js#L339). There is a slight difference but it looks like it could be reused.